qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -254,20 +217,19 @@ def random_cliffords(
             return [Clifford(self.clifford_2_qubit_circuit(i), validate=False) for i in samples]
 
     def random_clifford_circuits(
-        self, size: int = 1, rng: Optional[Union[int, Generator]] = None
-    ) -> List[QuantumCircuit]:
-        """Generate a list of random clifford circuits.
-        Used for 3 or more qubits"""
-        if self.num_qubits > 2:
-            return [random_clifford(self.num_qubits, seed=rng).to_circuit() for _ in range(size)]
+        self, num_qubits: int, size: int = 1, rng: Optional[Union[int, Generator]] = None
+    ):
+        """Generate a list of random clifford circuits"""
+        if num_qubits > 2:
```

Comment:
Sorry, I don't catch the point of your suggestion. I've just restored the original code before #892 here. Did you find any issue in the original code?

Reply:
It is not a big issue. I just meant the ordering of the `if` statements. It seems more natural to me to write:

```
if n == 1:
    do 1
if n == 2:
    do 2
else:
    do all other cases
```

Reply:
Ah, looking around it, I found two bugs in `ClifforUtils.random_cliffords` and `ClifforUtils.random_clifford_circuits` in the original code... I've fixed them in [01722e2](https://github.com/Qiskit/qiskit-experiments/pull/940/commits/01722e2dbfa3043c835cfaa59a2b9913829adf99) and decided to deprecate them since they are not used anywhere at least within qiskit-experiments.